examples/text_labels_and_annotations/tex_demo.py

```diff
@@ -3,17 +3,15 @@
 Rendering math equation using TeX
 =================================
 
-You can use TeX to render all of your matplotlib text if the rc
-parameter ``text.usetex`` is set.  This works currently on the agg and ps
-backends, and requires that you have tex and the other dependencies
-described in the :doc:`/tutorials/text/usetex` tutorial
-properly installed on your system.  The first time you run a script
-you will see a lot of output from tex and associated tools.  The next
-time, the run may be silent, as a lot of the information is cached.
-
-Notice how the label for the y axis is provided using unicode!
+You can use TeX to render all of your Matplotlib text by setting
+:rc:`text.usetex` to True.  This requires that you have TeX and the other
+dependencies described in the :doc:`/tutorials/text/usetex` tutorial properly
+installed on your system.  Matplotlib caches the processed TeX expressions, so
+if you run a script multiple times, runs after the first one will be faster.
```

Comment:
```suggestion
if you run a script multiple times, subsequent runs will be faster.
```
?

Reply:
that was my initial wording as well, but I think it's not completely clear "subsequent to what"?

Reply:
What about:

> Matplotlib caches processed TeX expressions, so that only the first usage of of an expression triggers a TeX compilation. Later usages reuse the rendered image from the cache and are thus faster.

Reply:
sure